extensions/amp-next-page/0.1/amp-next-page.js

```diff
@@ -57,22 +61,36 @@ export class AmpNextPage extends AMP.BaseElement {
     }
 
     const {element} = this;
-
     element.classList.add('i-amphtml-next-page');
 
-    // TODO(peterjosling): Read config from another source.
-
-    const scriptElements = childElementsByTag(element, 'SCRIPT');
-    user().assert(scriptElements.length == 1,
-        `${TAG} should contain only one <script> child.`);
-    const scriptElement = scriptElements[0];
-    user().assert(isJsonScriptTag(scriptElement),
-        `${TAG} config should ` +
-            'be inside a <script> tag with type="application/json"');
-    const configJson = tryParseJson(scriptElement.textContent, error => {
-      user().error(TAG, 'failed to parse config', error);
-    });
+    const src = element.getAttribute('src');
```

Comment:
My understanding is that this is introducing a new attribute. Needs:
- [ ] validator file and tests to be updated
- [ ] `documentation` to be updated

`amp-state` has a very similar feature for config (src or inline), would be a good reference.

Reply:
Validator file/tests updated.

I've got docs ready to go (https://github.com/emarchiori/amphtml/commit/2268125bca7e99d8a3cfc25f465625761312cf49) - should they go in this PR too, or I can wait to merge until this change makes it to canary or prod?

Docs are pending https://github.com/ampproject/amphtml/pull/16645 being merged as well.

Reply:
merging it after it makes it to prod is definitely better.

Reply:
The docs you linked to specify that `src` is required, however it also says it can be omitted. Please remove `(required)` if it can be omitted.

Reply:
Done